src/QirRuntime/README.md

```diff
@@ -36,26 +36,29 @@ CI builds and tests are enabled for this project. The build has no external depe
 ### Windows pre-reqs
 
 1. Install Clang, Ninja and CMake from the public distros.
-2. Add all three to your/system `%PATH%`.
-3. Install VS 2019 and enable "Desktop development with C++" component (Clang uses MSVC's standard library on Windows).
-4. Install clang-tidy and clang-format if your Clang/LLVM packages didn't include the tools.
-5. <_optional_> To use build/test scripts install Python 3.8.
+1. Add all three to your/system `%PATH%`.
+1. Install VS 2019 and enable "Desktop development with C++" component (Clang uses MSVC's standard library on Windows).
+1. Install clang-tidy and clang-format if your Clang/LLVM packages didn't include the tools.
+1. Install the same version of dotnet as specified by qsharp-runtime [README](../../README.md)
+1. <_optional_> To use build/test scripts install Python 3.8.
 
 *Building from Visual Studio and VS Code is **not** supported.
 Running cmake from the editors will likely default to MSVC or clang-cl and fail.*
 
 ### Linux via WSL pre-reqs
 
-1. On the host Windows machine [enable WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and install Ubuntu 20.04 LTS.
-2. In the Ubuntu's terminal:
+1. On the host Windows machine [enable WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and install
+ Ubuntu 20.04 LTS.
+1. In the Ubuntu's terminal:
     1. `$ sudo apt install cmake` (`$ cmake --version` should return 3.16.3)
-    2. `$ sudo apt-get install ninja-build` (`$ ninja --version` should return 1.10.0)
-    3. `$ sudo apt install clang` (`$ clang++ --version` should return 10.0.0)
-    4. Set Clang as the preferred C/C++ compiler:
+    1. `$ sudo apt-get install ninja-build` (`$ ninja --version` should return 1.10.0)
+    1. `$ sudo apt install clang` (`$ clang++ --version` should return 10.0.0)
+    1. Set Clang as the preferred C/C++ compiler:
         - $ export CC=/usr/bin/clang
         - $ export CXX=/usr/bin/clang++
```

Comment:
I would place these `$ export ...` commands _last_, after everything is installed (these `$ export` commands are not required to install `clang-tidy` below, right?).
The users typically install everything needed first, and then customize their *profile* scripts, to where these `$ export` commands will go.

Reply:
Well, I might not be a typical user, but I like dealing with groups of related tasks: install clang + set it to be the default, move on to installing other stuff... The thing I care much more about in this case is whether this export is at all necessary. When I wrote the instructions last spring I was running into issues with CMake refusing to pick up clang even if I passed it via the CMake's cmd compiler arg. We should try this again and, ideally, eliminate the requirement for export, which is too invasive... but this is outside of the scope of this PR.